src/main/cpp/mechanisms/DragonTale/ReadyState.cpp

```diff
@@ -76,6 +76,6 @@ bool ReadyState::AtTarget()
 bool ReadyState::IsTransitionCondition ( bool considerGamepadTransitions )
 {
 	// To get the current state use m_mechanism->GetCurrentState()
-	return false;
-	// return (considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::EXAMPLE_MECH_FORWARD));
+
+	return (considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::READY) || (m_mechanism->GetAlgaeSensorState() && m_mechanism->GetCoralInSensorState() && m_mechanism->GetCoralOutSensorState() && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::HUMAN_PLAYER_STATION)));
```

Comment:
NOT's(!) need to be added for the second set of conditions
Return True If any of these statements are True:

Ready Button is pressed
**Algae Sensor = False AND Coral Sensor In = False AND Coral Sensor Out = False AND Human Player Button is NOT pressed**